src/ElmSharp/ElmSharp/Transit.cs

```diff
@@ -246,7 +246,18 @@ public double EndAccelerationFactor
         /// <since_tizen> preview </since_tizen>
         public void Go(double interval = 0)
         {
-            Interop.Elementary.elm_transit_go_in(_handle, interval);
+            Double timeInterval = interval;
```

Comment:
```suggestion
```

Reply:
Thank you for review. I have updated the patch.